gap/oper.gi

```diff
@@ -63,8 +63,12 @@ end);
 
 InstallMethod(DigraphAddVertices, "for an immutable digraph and list",
 [IsImmutableDigraph, IsList],
-{D, labels} -> MakeImmutable(DigraphAddVertices(DigraphMutableCopy(D),
-                                                labels)));
+function(D, labels)
+  if IsEmpty(labels) then
```

Comment:
I know you didn't name this in your PR, but why is the argument named `labels`?

Reply:
Because this method adds `Length(labels)` new vertices, where the `i`th vertex to be added is given the label `labels[i]`.

Reply:
Makes sense, thanks!